File: howooyeon/bruet_force/카펫.java
class Solution {
  public int[] solution(int brown, int yellow) {
      int[] answer = new int[2];
      int y = 1;
      while(true){
          int x = brown/2 - y + 2;
              if(x * y - brown == yellow){
                  return new int[] {x, y};
              }
          y++;
      }
  }
}

File: devjaeyoon/brute_force/카펫.js
function solution(brown, yellow) {
  const answer = [];
  const total = brown + yellow;

  for (let i = 3; i <= brown / 2; i++) {
    if (total % i !== 0) {
      continue;
    }
    if ((total / i) * (i - 2) - (i - 2) * 2 === yellow) {
      answer.push([i, total / i]);
    }
  }

  return answer[answer.length - 1];
}
